Comparing /root/./workspace/apache/apex-core/385878814/.m2/org/apache/apex/apex-api/3.6.0/apex-api-3.6.0.jar with /root/workspace/apache/apex-core/385878814/api/target/apex-api-4.0.0-SNAPSHOT.jar:
**** MODIFIED ANNOTATION: PUBLIC ABSTRACT com.datatorrent.api.annotation.OperatorAnnotation  (not serializable)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) com.datatorrent.api.Operator$RecoveryMode recoveryMode()
***  MODIFIED CLASS: PUBLIC com.datatorrent.api.Attribute  (compatible)
	+++  NEW METHOD: PUBLIC(+) java.lang.String getLongName()
***  MODIFIED INTERFACE: PUBLIC ABSTRACT STATIC com.datatorrent.api.Context$DAGContext  (not serializable)
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) com.datatorrent.api.Attribute LOGGER_APPENDER
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) com.datatorrent.api.Attribute SSL_CONFIG
***  MODIFIED INTERFACE: PUBLIC ABSTRACT STATIC com.datatorrent.api.Context$OperatorContext  (not serializable)
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) com.datatorrent.api.Attribute RECOVERY_MODE
**** MODIFIED INTERFACE: PUBLIC ABSTRACT STATIC com.datatorrent.api.DAG$OperatorMeta  (default serialVersionUID changed)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) com.datatorrent.api.annotation.OperatorAnnotation getOperatorAnnotation()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT com.datatorrent.api.StatsListener  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT com.datatorrent.api.StatsListener$Response processStats(com.datatorrent.api.StatsListener$BatchedOperatorStats)
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++  NEW ANNOTATION: java.lang.Deprecated
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) STATIC(+) com.datatorrent.api.StatsListener$StatsListenerContext  (not serializable)
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.String getOperatorName(int)
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) STATIC(+) com.datatorrent.api.StatsListener$StatsListenerWithContext  (not serializable)
	+++  NEW INTERFACE: com.datatorrent.api.StatsListener
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) com.datatorrent.api.StatsListener$Response processStats(com.datatorrent.api.StatsListener$BatchedOperatorStats, com.datatorrent.api.StatsListener$StatsListenerContext)
